channel.json

```diff
@@ -32,6 +32,7 @@
 		"https://raw.githubusercontent.com/damccull/sublimetext-SolarizedToggle/master/packages.json",
 		"https://raw.githubusercontent.com/danielmagnussons/orgmode/master/packages.json",
 		"https://raw.githubusercontent.com/danielobrien/sublime-DLX-syntax/master/packages.json",
+		"https://raw.githubusercontent.com/devilthrone/WeexTemplate/master/packages.json",
```

Comment:
This should be deleted, along with the file you've referenced here. The entry in w.json it what will add it to the default channel.